mcp/include/labjack.h

```diff
@@ -36,7 +36,7 @@
 
 #define LABJACK_OF_POWER 0
 #define LABJACK_IF_POWER 1
-#define LABJACK_OF_1 2
+#define LABJACK_MOTOR_POWER 2
```

Comment:
Is LABJACK_OF_1 being used elsewhere? Does redefining this pull the rug out from under another part of the code?

Reply:
Looks like it's just stuff in `legacyMCP` dir, so we should be good.

Reply:
its from legacyMCP stuff thats why I stole it and remapped it.